_layouts/roadmap.html

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+---
+<section class="hero-section section hero-roadmap">
+    <div class="container hero-section-content-centered">
+        <div class="hero-spotlight-header hero-spotlight-header-page">
+            <h1>Foreseen development in Indico</h1>
```

Comment:
"Planned"?

Or maybe just call it what it is - "Roadmap"

Reply:
I agree that "roadmap" would be simpler, but having short header doesn't look good visually. I'll change it to "planned"